public/gdc/assets/sass/areas/_home.scss

```diff
@@ -0,0 +1,744 @@
+// Base
+// --------------------------------------------------
+
+html {
+  background: #0253A7 url(../../img/bg_grain.png);
+  border-top: 2px solid #fff;
+}
+body {
+  background: url(../../img/bg_top@2x.png) 50% -70px no-repeat;
+  background-size: 1920px auto;
+}
+.section-title {
+  text-transform: uppercase;
+  color: #fff;
+  letter-spacing: 5px;
+  font-size: 1.8rem;
+}
+
+
+// Header
+// --------------------------------------------------
+
+.header {
+  background: transparent;
+  text-align: center;
+  padding-top: 0;
+  margin-top: -50px;
+  img {
+    max-width: 90%;
+    display: block;
+    margin: 190px auto 0;
+    font-size: 8rem;
+    @include animation(fadein 3s);
+    &.headline-mobile {
+      display: none;
+    }
+  }
+}
+
+
+// Announcement
+// --------------------------------------------------
+
+.announcement {
+  text-align: center;
+  margin: 160px auto 60px;
+  max-width: 850px;
+  display: none;
+  .announcement-active & {
+    display: block;
+  }
+  .message {
+    background: rgba(16, 145, 206, 0.25);
+    padding: 50px 50px 60px;
+    border-radius: 5px;
+    h4 {
+      color: #FFF;
+      font-size: 2.1rem;
+      font-style: italic;
+      & strong {
+        font-style: normal;
+        font-weight: 700;
+        font-size: 2.4rem;
+        text-transform: uppercase;
+      }
+    }
+    p {
+      padding: 3px 0;
+      font-size: 2.0rem;
+      line-height: 1.3;
+      max-width: 80%;
+      margin: 0 auto;
+    }
+    strong {
+      font-weight: 600;
+    }
+  }
+  .btn {
+    background-color: rgb(8, 66, 119);
+    outline: none;
+    border: none;
+    color: #1091CE;
+    padding: 24px 42px;
+    border-radius: 15px;
+    letter-spacing: 3.6px;
+    font-weight: 600;
+    font-family: $open;
+    font-size: 1.3rem;
+    margin: -32px auto 0;
+    text-transform: uppercase;
+    position: relative;
+    z-index: 1;
+    @include transition(all .3s);
+    &:hover {
+      color: #FFF;
+      background-color: rgb(48, 106, 159);
+    }
+  }
+}
+
+
+// Top Stories
+// --------------------------------------------------
+
+.top-stories {
+  margin-top: 60px;
+  body:not(.announcement-active) & {
+    margin-top: 160px;
+  }
+  h4 {
+    margin-left: 15px;
+    margin-bottom: 30px;
+    body:not(.announcement-active) & {
+      text-align: center;
+    }
+  }
+  .wrapper {
+    @include make-xs-column(12);
+    @include make-sm-column(6);
+    &.wrapper-landscape {
+      @include make-sm-column(12);
+    }
+  }
+}
+
+.wrapper {
+  margin-bottom: 28px;
+  padding: 0 14px;
+}
+
+.link {
+  position: relative;
+  margin: 0 0 5px;
+  display: block;
+  color: $text-color;
+  text-decoration: none;
+  overflow: hidden;
+  box-shadow: rgba(0, 0, 0, 0.3) 0px 0px 10px;
+  background-color: #fff;
+  -webkit-transform: translateZ(0px);
```

Comment:
is this just to GPU accelerate?

also, this should be `transform: translateZ(0);` - not vendor prefixed.